Doc/library/sys.rst

```diff
@@ -1060,6 +1062,31 @@ always available.
         This is still called as a fallback if a :data:`meta_path` entry doesn't
         have a :meth:`~importlib.abc.MetaPathFinder.find_spec` method.
 
+.. data:: module_names
```

Comment:
Question: as the main use of this is checking if something is a stdlib module we can return a frozenset here. Any reason the order is important?

Reply:
I'm used to tuple, I have no preference between tuple and frozenset. It can be a frozenset.

Reply:
I would prefer a frozenset, just because the main use is comparison. We can leave the actual type not specify and say that is an "iterable"